### PR TITLE
Gci 1364 add certified copy to private sdk api

### PR DESCRIPTION
**Pull request by rjadhav-ch**
This adds changes to data mapping that result from updates to private api sdk.

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/orders/api/listener/OrderItemOptionsReader.java [hunk 1]
```diff
@@ -20,29 +21,13 @@
 class OrderItemOptionsReader {
 
     private final ObjectMapper mapper;
+    private static final String CERTIFICATE_ITEM_OPTIONS_TYPE = "CertificateItemOptions";
+    private static final String CERTIFIEDCOPY_ITEM_OPTIONS_TYPE = "CertifiedCopyItemOptions";
```
Comment (nabeelamirch) [inline src/main/java/uk/gov/companieshouse/orders/api/listener/OrderItemOptionsReader.java:25]: I'm guessing you don't need this anymore

Comment (rjadhav-ch) [reply src/main/java/uk/gov/companieshouse/orders/api/listener/OrderItemOptionsReader.java]: Thanks. It's been removed.

File: src/main/java/uk/gov/companieshouse/orders/api/mapper/ApiToItemMapper.java [hunk 2]
```diff
@@ -1,37 +1,60 @@
...
+    @Mapping(source = "descriptionIdentifier", target="descriptionIdentifier")
+    @Mapping(source = "descriptionValues", target="descriptionValues")
+    @Mapping(source = "itemCosts", target="itemCosts")
+    @Mapping(source = "etag", target="etag")
+    @Mapping(source = "kind", target="kind")
+    @Mapping(source = "postalDelivery", target="postalDelivery")
+    @Mapping(source = "quantity", target="quantity")
+    @Mapping(source = "links", target="links")
+    @Mapping(source = "postageCost", target="postageCost")
+    @Mapping(source = "totalItemCost", target="totalItemCost")
```
Comment (VaughanJackson123) [inline src/main/java/uk/gov/companieshouse/orders/api/mapper/ApiToItemMapper.java:34]: Why are all of these mappings required?

Comment (rjadhav-ch) [reply src/main/java/uk/gov/companieshouse/orders/api/mapper/ApiToItemMapper.java]: Yes, they are unnecessary. Thanks, I have removed them.

**Review by nabeelamirch — APPROVED** [review]

**Review by VaughanJackson123 — APPROVED** [review]

**Pull request merged**